python-package/lightgbm/dask.py

```diff
@@ -604,7 +615,23 @@ def fit(
             **kwargs
         )
 
-    fit.__doc__ = LGBMClassifier.fit.__doc__
+    _base_doc = _lgbmmodel_doc_fit.format(
+        X_shape="dask Array or dask DataFrame of shape = [n_samples, n_features]",
+        y_shape="dask Array, dask DataFrame or dask Series of shape = [n_samples]",
+        sample_weight_shape="dask Array, dask DataFrame, Dask Series of shape = [n_samples] or None, optional (default=None)",
+        group_shape="dask Array, dask DataFrame, Dask Series of shape = [n_samples] or None, optional (default=None)"
```

Comment:
```suggestion
        sample_weight_shape="dask Array, dask DataFrame, dask Series of shape = [n_samples] or None, optional (default=None)",
        group_shape="dask Array, dask DataFrame, dask Series of shape = [n_samples] or None, optional (default=None)"
```

Reply:
@StrikerRUS I just checked the Dask docs for guidance, and I think based on https://marketing.dask.org/en/latest/refer.html that we should actually prefer capitalized `Dask` when talking about the project, and lowercase `dask` when referring to the specific Python package `dask`. So for example:

* `Dask DataFrame`
* `dask.distributed client`

I'll make that change everywhere in this PR, including in lines not currently touched by it. I think it makes sense in the scope of this PR.

h/t to @rikturr for pointing this out to me (https://github.com/dask/dask.github.io/pull/44)

Reply:
OK, thank you!
We don't have a lot of docs about Dask yet, so I think it is OK to do it right here in this PR.

Reply:
updated this everywhere I could find in https://github.com/microsoft/LightGBM/pull/3930/commits/8cceecf2f0b184f22559da58c8c12891e8bcd6e3

I used `git grep -i dask` from the root of the repo to check for other cases and don't see any that need their capitalizations changed

Reply:
Great! Many thanks!